pyomo/contrib/mindtpy/util.py

```diff
@@ -42,6 +42,7 @@ class MindtPySolveData(object):
     """
     pass
 
+# TODO: move all functions in this file to algorithm base class and solve_data data can be removed.
```

Comment:
Yes

Reply:
Maybe not all functions. The functions that applies to all strategys and do not depend on solve_data can be kept here.